Dockerfile.dl-glove-6B-50d

```diff
@@ -1,6 +1,6 @@
-FROM elifesciences/sciencebeam-trainer-delft-grobid_unstable:04346b26b5fa7966cef4d6ec6bba867feba52d66-dee3b060495006aa9f8e1b89f6278b44231fe0cc
+ARG base_image
+FROM ${base_image}
 
-ENV GROBID_MODELS_DIRECTORY=/opt/grobid/grobid-home/models
 ENV EMBEDDING_REGISTRY_PATH="${PROJECT_FOLDER}/embedding-registry.json"
```

Comment:
guess `{PROJECT_FOLDER}`  is defined in some other base image

Reply:
Yes, it is defined in the base image.